src/components/MobileNavigation/MobileNavigation.scss

```diff
@@ -0,0 +1,42 @@
+.MobileNavigation__wrapper {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 32px;
+    gap: 50px;
+    height: 798px;
+    left: 0px;
+    top: 96px;
+    background: var(--color-white-lilac);
+    border-bottom: 1px solid #e3e0ed;
+
+    &__link1, &__link2 {
+        font-style: normal;
+        font-weight: 600;
+        font-size: 14px;
+        line-height: 17px;
+        color: var(--color-bunting);
+        flex: none;
+        order: 0;
+        flex-grow: 0;
+    }
+
+    &__button {
+        display: flex;
+        flex-direction: row;
+        justify-content: center;
+        align-items: center;
+        padding: 15.5px 24px;
+        gap: 10px;
+        width: 326px;
```

Comment:
change width to like 100% to make responsive?

Reply:
I would also remove all flex properties here, but add text-decoration: none; and maybe text-alight: center;


btw, var(--color-white) should be used instead of #FFFFFF